common/iterator/sorted/SortedIterators.java

```diff
@@ -116,6 +116,15 @@
             return new MergeMappedIterator.Forwardable<>(iterators, e -> e, order);
         }
 
+        @SafeVarargs
```

Comment:
SafeVarargs is used to indicate to the Java compiler that a signature like this is okay:
```
<T> T[] toArray(T... elements) { }
```
this is a a warning from Java because arrays cannot be generified.